src/components/NotificationsAlert.tsx

```diff
@@ -30,6 +31,8 @@ export const NotificationsAlert = () => {
   const { isIosDevice, versionOfIos } = useIosDevice();
   const { showOnTS, subscribeStarted, subscribeError } = getNotificationLocalStorage() || {};
   const tosData = useTermsOfServiceStore((store) => store.tosData);
+  const showModalOnTs = useMemo(() => showOnTS, [showOnTS]);
+  const subscriptionError = useMemo(() => subscribeError, [subscribeError]);
```

Comment:
I believe this usage of `useMemo()` isn't really achieving anything. Wouldn't it be the same as using `showOnTS` and `subscribeError` directly?

Reply:
Yeah, I forgot to remove that. The t was a part of my testing